extensions/amp-selector/0.1/amp-selector.js

```diff
@@ -95,11 +95,16 @@ export class AmpSelector extends AMP.BaseElement {
     if (!this.isMultiple_) {
       selectedArray = selectedArray.slice(0, 1);
     }
+    // Convert array values to strings and create map for fast lookup.
+    const selectedMap = selectedArray.reduce((map, value) => {
+      map[value] = true;
+      return map;
+    }, Object.create(null));
```

Comment:
String conversion is necessary to compare to elements' `option` attribute values. If we need to convert to string, might as well use a map for fast lookups.